internal/blockchain/ethereum/ethereum.go

```diff
@@ -32,8 +32,10 @@ import (
 )
 
 type Account struct {
-	Address    string `json:"address"`
-	PrivateKey string `json:"privateKey"`
+	Address       string `json:"address"`
+	PrivateKey    string `json:"privateKey"`
```

Comment:
For our use case, only the public Tessera key is necessary. Applications wanting to share private payloads need to know which Tessera to route to.

We weren't too sure why the Blockchain signer keys are exposed in this data structure, so we went to mirror the same structure for Tessera. I'll remove the Tessera private key from this structure.

There are more complicated setup for Tessera where keys need to be instantiated in Hashicorp Vault. For purpose of the development environment, we want to do away with these complexities as well.

Reply:
Cool - that's great @rodion-lim-partior 

That structure is used for building the `stack.json` file - when I (or @EnriqueL8) come back round on this one, I think we need to do a bit of digging into the existing use to see why `privateKey` is there at all already. It might just be for reference.

The `CreateAccount` in the signer is the main place this is updated, when a new account is added - but that's after it's copied to the volume.

https://github.com/hyperledger/firefly-cli/blob/1378d838891cd0a43d9a255dac9b707a69f13c59/internal/blockchain/ethereum/ethsigner/ethsigner.go#L207-L210